payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/PayrollData.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace PayrollProcessor.Functions.Features.Payroll
+namespace PayrollProcessor.Functions.Features.Payrolls
```

Comment:
I changed the namespaces to plural to prevent conflicts between class names and namespace names

See: https://stackoverflow.com/a/18731452/939634

Reply:
Yeah they should have been from the start, i run into this issue all the time with singular namespaces